.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = F401,F403
```

Comment:
The `F401` and `F403` are rules that make sure you are using all of your imports. Some files, like `__init__.py`, have named imports that aren't being used but were tripping up these rules. I was worried about breaking something by removing those so for now I'm just ignoring this rule.